### PR TITLE
Support for HDP

### PR DESCRIPTION
**Pull request by SparshSinghalHM**
Added parameter for CISF protection in Ambari. Can be provided in a separate branch.

### CODE REVIEW

File: service/src/main/java/org/apache/griffin/core/job/SparkSubmitJob.java [hunk 1]
```diff
@@ -113,8 +117,13 @@ private void updateJobInstanceState(JobExecutionContext context) throws
     private String post2Livy() {
         String result = null;
         try {
-            result = restTemplate.postForObject(livyUri, livyConfMap,
-                    String.class);
+            HttpHeaders headers = new HttpHeaders();
+            headers.setContentType(MediaType.APPLICATION_JSON);
+            headers.set("X-Requested-By","admin");
```
Comment (xuspi) [inline service/src/main/java/org/apache/griffin/core/job/SparkSubmitJob.java]: Could you use a constant for the "X-Requested-By"?
  public static final String REQUEST_BY_HEADER = "X-Requested-By";

**Comment by bhlx3lyx7** [comment]
I've tested the code, it works in docker.
"X-Requested-By" could be set as any value, is it appropriate to hardcode it as "admin"?

File: service/src/main/java/org/apache/griffin/core/job/SparkSubmitJob.java (hunk 1 above)
Comment (SparshSinghalHM) [reply service/src/main/java/org/apache/griffin/core/job/SparkSubmitJob.java]: Yes. Can do it this way.

**Comment by SparshSinghalHM** [comment]
@bhlx3lyx7  No, not required. We can ask for the ambari username in application.properties. Admin is the default user.

**Pull request closed without merging**